recipes/apache-flink/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "apache-flink" %}
+{% set version = "1.18.0" %}
+
+package:
+  name: "apache-flink-python"
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/apache-flink-{{ version }}.tar.gz
+  sha256: 071e7804c87a01e48d69f6ad96e8f85256267acc9788e5261ace06158f2b7f95
+
+build:
+  skip: true  # [py<37 or win]
```

Comment:
Should this skip Windows? Although, it would then require the `pemja` and `apache-flink-libraries` to also be built for Windows.

Reply:
yeah, I'll leave that to the next intrepid packager.  I think it might require upstream changes in `pemja`